### ISSUE TITLE
Receiving "The property 'OdataType' does not exist on type 'microsoft.graph.message'." despite not passing OdataType to Message

### ISSUE THREAD

**Issue by caelan-schneider** [microsoftgraph/msgraph-sdk-python#440]
I started receiving this error today. My code has worked for the past two months without issue, and I haven't made any changes or updated my msgraph-sdk version. I'm not specififying OdataType anywhere in my Message object: 

```
message=Message(
    subject=subject,
    body=ItemBody(content_type=BodyType.Html, content=html),
    to_recipients=recipients,
    attachments=images
)
```

Error:
```
msgraph.generated.models.o_data_errors.o_data_error.ODataError: 
        APIError
        Code: 400
        message: None
        error: MainError(additional_data={}, code='RequestBodyRead', details=None, inner_error=None, message="The property 'OdataType' does not exist on type 'microsoft.graph.message'. Make sure to only use property names that are defined by the type or mark the type as open type.", target=None)
```

**Comment by caelan-schneider** [comment]
specifying odata_type="#microsoft.graph.message" in the Message object resolves this, but the solution seems odd given that "#microsoft.graph.message" is the default parameter for odata_type in the Message object.

**Comment by samwelkanda** [comment]
Hi @caelan-schneider 

Most likely this is caused by updates to our serialization library. 

We have recently released a stable version of the SDK (v1.0.0) that I highly recommend you upgrade to and will resolve the said issue.

Thanks
